src/spyglass/utils/dj_mixin.py

```diff
@@ -131,6 +137,8 @@ def _safe_context(cls):
             else nullcontext()
         )
 
+    _parallel_make = False
```

Comment:
I would move this to either the set of attrs just before the init or just before the new `populate`